theme/src/theme/DocBanner/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { useActivePlugin } from '@docusaurus/plugin-content-docs/client';
+import { useDoc } from '@docusaurus/theme-common/internal';
+import { DocMetadata } from '@iota-wiki/plugin-docs';
+import ReactMarkdown from 'react-markdown';
+import './styles.css';
+
+export default function DocBanner() {
+  const { bannerContent } = useDoc().metadata as DocMetadata;
+
+  if (!bannerContent) return null;
+
+  return (
+    <ReactMarkdown className='doc-banner alert alert--warning margin-bottom--md'>
```

Comment:
It's not necessary for this version, but it may be nice to be able to choose the type/class of the banner. WDYT?

Reply:
Yeah I've been thinking about that. We could easily add that function, but for this iteration I wanted to stay as close to the Docusaurus version banner as possible.

Reply:
Could you create an issue for that?

Reply:
Tracked by #1106